### PR TITLE
FEAT: Add option to choose LP solver to repeated_game.jl

### LINKED ISSUE #58: outerapproximation: Add option to choose LP solver
Right now [`ClpSolver ` is hard coded](https://github.com/QuantEcon/Games.jl/blob/113fea9040325171812d5f6675bd9bbb03947e05/src/repeated_game.jl#L368).

### PR DESCRIPTION
**Pull request by QBatista**
Adds an option to choose an LP solver for repeated_game.jl

Close #58

### CODE REVIEW

File: src/repeated_game.jl [hunk 1]
```diff
@@ -221,19 +221,23 @@ end
...
+   GLPKSolverLP() and GurobiSolver(). By default, it choooses ClpSolver().
+
 
 # Returns
 
 - `out::Float64` : Worst possible payoff for player i.
 """
 function worst_value_i(rpd::RepGame2, H::Array{Float64, 2},
-                       C::Array{Float64, 1}, i::Int)
+                       C::Array{Float64, 1}, i::Int, lp_solver=ClpSolver())
```
Comment (oyamad) [inline src/repeated_game.jl]: Add a type declaration. (Several places.)

Comment (QBatista) [reply src/repeated_game.jl]: Should I add and use  `AbstractMathProgSolver` as in `linprog`?

Comment (oyamad) [reply src/repeated_game.jl]: Yes. Thanks.

**Comment by oyamad** [comment]
Maybe set a default (`ClpSolver()`) to `lp_solver` everywhere?

**Pull request merged**